packages/kit/test/apps/basics/test/test.js

```diff
@@ -234,6 +234,18 @@ test.describe('Scrolling', () => {
 		expect(await in_view('#input')).toBe(true);
 		expect(await page.$eval('#input', (el) => el === document.activeElement)).toBe(true);
 	});
+
+	test.only('scroll to top works with overflowing root container', async ({ page, clicknav }) => {
+		await page.goto('/scroll-top/overflow');
```

Comment:
I feel like `scroll-top` isn't super descriptive about what this is testing because we have other scroll tests that test scrolling to the top. How about something like `scroll-overflowed`?